packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -0,0 +1,13516 @@
+// @public (undocumented)

```

Comment:
We recently upgraded our version of api-extractor and are now tracking api changes in the *.api.md files (which you have above). So, we want to avoid committing the old *.api.ts. If you could delete this from your branch that would be much appreciated! :)

Reply:
updated.